src/components/protocol_handler/src/protocol_handler_impl.cc

```diff
@@ -2046,7 +2066,17 @@ RESULT_CODE ProtocolHandlerImpl::EncryptFrame(ProtocolFramePtr packet) {
       packet->connection_id(), packet->session_id());
   security_manager::SSLContext* context = session_observer_.GetSSLContext(
       connection_key, ServiceTypeFromByte(packet->service_type()));
-  if (!context || !context->IsInitCompleted()) {
+
+  LOG4CXX_DEBUG(logger_,
+                "Protection flag is: " << packet->protection_flag()
+                                       << std::boolalpha);
+  LOG4CXX_DEBUG(logger_,
+                "RPC service for connection id: "
```

Comment:
@BSolonenko i think this log message is redundant

Reply:
@mked-luxoft removed